src/index.js

```diff
@@ -0,0 +1,58 @@
+import './style.css';
+import './cute.png';
+import './anime.jpeg';
+import fetchAnime from './modules/fetchAnimes.js';
+import displayAnime from './modules/displayMovies.js';
+import postLike from './modules/postLikes.js';
+import getLikes from './modules/getLikes.js';
+import showLike from './modules/showLikes.js';
+import moviePopup from './modules/moviePopup.js';
+
+const showPopup = document.querySelector('.popup');
+
+const likedAnimes = async () => {
+  const likeBtn = document.querySelectorAll('.btn-like');
+  likeBtn.forEach((btn) => {
+    btn.addEventListener('click', async () => {
+      await postLike(btn.dataset.id);
+      const likes = await getLikes();
+      showLike(likes);
+    });
+  });
+};
+
+window.addEventListener('load', async () => {
+  const animes = await fetchAnime();
+  displayAnime(animes);
+  likedAnimes();
+  const likes = await getLikes();
+  showLike(likes);
+
+  const movies = document.querySelectorAll('.anime-holder');
+  const allMovies = movies.length;
+  const totalHolder = document.querySelector('.totality');
+  totalHolder.innerHTML = allMovies;
```

Comment:
- [ ] This is the counter functionality that counts the number of movies/series being displayed on the Home page. This function works fine, but it should be implemented in a separate module so you can import it into your test file. Kindly create a new module for it so you can import it here and on your test file.